scripts/bash_aliases.sh

```diff
@@ -8,7 +8,7 @@
 
 
 # Directory navigation
-alias sub="cd $CATKIN_DIR/src/SubjuGator"
+alias sub="cd \$CATKIN_DIR/src/SubjuGator"
```

Comment:
The change escapes the `$` character, which means the variable's value will be based on when the alias is used rather than when it is set.